Formula/byobu.rb

```diff
@@ -1,6 +1,6 @@
 class Byobu < Formula
   desc "Text-based window manager and terminal multiplexer"
-  homepage "http://byobu.co/"
+  homepage "https://github.com/dustinkirkland/byobu"
```

Comment:
github repo is a mirror, not the official upstream

Reply:
The repository is listed on the original page:
https://web.archive.org/web/20190509202352/byobu.co/source.html
![Screenshot 2019-07-20 at 20 20 51](https://user-images.githubusercontent.com/1446897/61582500-027a7000-ab2c-11e9-9cdd-9d79efaf5d36.png)

Reply:
But the repo clearly states:

> git mirror of upstream bzr at http://launchpad.net/byobu

and the launchpad page has a lot more info, so it makes sense for us to use it

Reply:
Okay!